chapter2/tests/t19/terraform/_modules/nginx/main.tf

```diff
@@ -0,0 +1,31 @@
+module "container" {
+  source                       = "../container"
+  container_image              = var.container_image
+  container_image_keep_locally = var.container_image_keep_locally
+  container_name               = var.container_name != null ? var.container_name : ""
```

Comment:
make all validation in the inner module, not in the outer invocation. That way if your module REQUIRE the name - you will get error

Reply:
> make all validation in the inner module, not in the outer invocation. That way if your module REQUIRE the name - you will get error

@dmitry-mightydevops
Done